src/main/resources/chinese.yml

```diff
@@ -0,0 +1,144 @@
+msg_err_cannot_nationcollect_not_in_capital: "&c你必须处在首都里面才可以收集国家资源."
+msg_err_cannot_nationcollect_no_resources_available: '&c当前国家没有可以收集的资源.'
+
+msg_err_cannot_collect_uknown_material: "&c未知物品类型: %s. 无法收集该资源."
+
+# Player Extraction Limits
+msg_error_daily_extraction_limit_reached: "&c资源 %s 已达到每日采集上限 (%d)"
+daily_extraction_limits_reset: "&b所有每日采集上限已重置."
+
+# Resource Categories
+# The format of each of these is:  'extraction name, discovery name'
+resource_category_common_dirt: '普通的泥土, 普通的泥土'
+resource_category_uncommon_dirt: '不同寻常的泥土, 不同寻常的泥土'
+resource_category_rare_dirt: '稀有的泥土, 稀有的泥土'
+resource_category_gravel: '沙砾, 沙砾'
+resource_category_sand: '沙子, 沙子'
+resource_category_wood: '木头, 木头'
+resource_category_common_rocks: '普通的岩石, 普通的岩石'
+resource_category_uncommon_rocks: '不同寻常的岩石, 不同寻常的岩石'
+resource_category_terracotta: '陶瓦, 陶瓦'
+resource_category_quartz: '石英, 石英'
+resource_category_prismarine: '海晶石, 海晶石'
+resource_category_obsidian: '黑曜石, 黑曜石'
+resource_category_clay: '黏土块, 黏土块'
+resource_category_ice: '冰, 冰'
+resource_category_snow: '雪, 雪'
+resource_category_coal: '煤, 煤'
+resource_category_iron: '铁, 铁'
+resource_category_gold: '金, 金'
+resource_category_gold_nugget: '金粒, 金子'
+resource_category_copper: '铜, 铜'
+resource_category_emeralds: '绿宝石, 绿宝石'
+resource_category_lapis_lazuli: '青金石, 青金石'
+resource_category_diamonds: '钻石, 钻石'
+resource_category_netherrack: '地狱岩, 地狱岩'
+resource_category_nether_quartz: '下届石英, 下届石英'
+resource_category_glowstone: '萤石, 萤石'
+resource_category_end_stone: '末地石, 末地石'
+resource_category_ancient_debris: '远古残骸, 远古残骸'
+resource_category_wool: '羊毛, 羊群'
+resource_category_coral: '珊瑚, 珊瑚'
+resource_category_wheat: '小麦, 肥沃的土壤'
+resource_category_carrots: '胡萝卜, 肥沃的土壤'
+resource_category_potatoes: '土豆, 肥沃的土壤'
+resource_category_beetroot: '甜菜, 肥沃的土壤'
+resource_category_pumpkin_seeds: '南瓜种子, 肥沃的土壤'
+resource_category_melon_slice: '西瓜片, 肥沃的土壤'
+resource_category_cocoa_beans: '可可豆, 肥沃的土壤'
+resource_category_pumpkin_blocks: '南瓜块, 肥沃的土壤'
+resource_category_melon_blocks: '西瓜块, 肥沃的土壤'
+resource_category_cocoa_pods: '可可豆荚, 肥沃的土壤'
+resource_category_kelp: '海藻, 海藻'
+resource_category_bamboo: '竹子, 肥沃的土壤'
+resource_category_cactus: '仙人掌, 肥沃的土壤'
+resource_category_mushrooms: '蘑菇, 蘑菇'
+resource_category_apples: '苹果, 苹果园'
+resource_category_vines: '藤蔓, 藤蔓'
+resource_category_nether_wart: '地狱疣, 地狱疣'
+resource_category_chorus_fruit: '紫颂果, 紫颂果'
+resource_category_fish: '鱼, 鱼群'
+resource_category_gunpowder: '火药, 遗弃的军火库'
+resource_category_string: '线, 盘丝洞'
+resource_category_spider_eyes: '蜘蛛眼, 蜘蛛巢'
+resource_category_feathers: '羽毛, 遗弃的养鸡场'
+resource_category_leather: '皮革, 一大群牛'
+resource_category_rabbit_hides: '兔子皮, 兔子洞'
+resource_category_rotten_flesh: '新鲜的腐肉, 尸潮'
+resource_category_bones: '骨头, 奇怪的骨骸'
+resource_category_pork: '猪肉, '
+resource_category_mutton: '羊肉, 羊群'
+resource_category_beef: '牛肉, 牛群'
+resource_category_chickens: '鸡肉, 遗弃的养鸡场'
+resource_category_eggs: '鸡蛋, 遗弃的养鸡场'
+resource_category_ender_pearls: '末影珍珠, 折跃失败的末影人'
+
```

Comment:
:construction: 
- Hi, just this morning, I added "slime" as a new resource category.
- For completeness of your file, could you add that translation also pls before we merge.

Reply:
I'm eager to get the (unrelated) slime fix out. So I'll say lets merge this now with just an english translation, and we can fix later